packages/bumbag/src/styled/index.ts

```diff
@@ -1,5 +1,5 @@
 export { default as classNames } from 'classnames';
-export { flush, hydrate, cx, getRegisteredStyles, injectGlobal, sheet, cache, css as cssClass } from 'emotion';
-export { css, keyframes, Global, ThemeContext, CacheProvider } from '@emotion/core';
-export { withTheme, ThemeProvider } from 'emotion-theming';
+export { flush, hydrate, cx, getRegisteredStyles, injectGlobal, sheet, css as cssClass, cache } from '@emotion/css';
+export { css, keyframes, Global, ThemeContext, CacheProvider } from '@emotion/react';
```

Comment:
We can probably consolidate these lines so `withTheme` and `ThemeProvider` is exported here too.

Reply:
Thanks for the catch!